condition/condition_plugins/src/speed_condition.cpp

```diff
@@ -53,9 +53,10 @@ bool SpeedCondition::update(
     {
       double npc_velocity { 0.0 };
 
-      if (not (*api_ptr_).getNPCVelocity(trigger_, &npc_velocity))
+      if (!(*api_ptr_).getNPCVelocity(trigger_, &npc_velocity))
       {
-        ROS_ERROR_STREAM("Invalid trigger name specified for " << getType() << " condition named " << getName());
+        // TODO(yunus.caliskan): use the logger of the simulator instead.
```

Comment:
Is this a different situation or can you just take the same logger as in the other similar case

Reply:
Fixed.